### ISSUE TITLE
Steam requires libSDL2-2.0.so.0 to be installed

### ISSUE THREAD

**Issue by zeroepoch** [negativo17/steam#9]
TL;DR I think there is a missing dependency on `SDL.i686`.

I don't know if this is a new issue since Fedora 40 (recently upgraded) or with Wayland, but when starting Steam under Wayland + NVIDIA r555 (from the negativo repo) `gldriverquery` crashes on Steam start up. It happens in the background so I only noticed it when looking under `/var/lib/systemd/coredump/`, and then noticing this in `journalctl`. While search around [someone mentioned](https://github.com/ValveSoftware/steam-for-linux/issues/8853#issuecomment-2039921854) running the binary under `gdb` to get a stack trace. I figured out at that point that `libSDL2-2.0.so.0` was missing. After `dnf install libSDL2-2.0.so.0` which installs `SDL2.i686` I was able to run the binary under `gdb` and it exited fine. I also no longer see core dumps or errors in the journal each time I start Steam after this.

FYI, here is what I get for a backtrace from `journalctl`
```
[🡕] Process 5145 (gldriverquery) of user 1000 dumped core.
Module libxcb-sync.so.1 from rpm libxcb-1.17.0-1.fc40.i386
Module libxcb-present.so.0 from rpm libxcb-1.17.0-1.fc40.i386
Module libX11-xcb.so.1 from rpm libX11-1.8.9-1.fc40.i386
Module libxcb-dri3.so.0 from rpm libxcb-1.17.0-1.fc40.i386
Module libxcb-randr.so.0 from rpm libxcb-1.17.0-1.fc40.i386
Module libxcb-glx.so.0 from rpm libxcb-1.17.0-1.fc40.i386
Module libdrm.so.2 from rpm libdrm-2.4.121-1.fc40.i386
Module libGLdispatch.so.0 from rpm libglvnd-1.7.0-4.fc40.i386
Module libGLX.so.0 from rpm libglvnd-1.7.0-4.fc40.i386
Module libGL.so.1 from rpm libglvnd-1.7.0-4.fc40.i386
Module libXext.so.6 from rpm libXext-1.3.6-1.fc40.i386
Module libXau.so.6 from rpm libXau-1.0.11-6.fc40.i386
Module libxcb.so.1 from rpm libxcb-1.17.0-1.fc40.i386
Module libX11.so.6 from rpm libX11-1.8.9-1.fc40.i386
Module libzstd.so.1 from rpm zstd-1.5.6-1.fc40.i386
Module liblzma.so.5 from rpm xz-5.4.6-3.fc40.i386
Module liblz4.so.1 from rpm lz4-1.9.4-6.fc40.i386
Module libcap.so.2 from rpm libcap-2.69-8.fc40.i386
Module libsystemd.so.0 from rpm systemd-255.8-1.fc40.i386
Stack trace of thread 5145:
#0  0x00000000f76dc33a xcb_send_request_with_fds64 (libxcb.so.1 + 0x1333a)
#1  0x00000000f76dcad8 xcb_send_request64 (libxcb.so.1 + 0x13ad8)
#2  0x00000000f76dcb04 xcb_send_request (libxcb.so.1 + 0x13b04)
#3  0x00000000f19729ae xcb_dri3_free_syncobj (libxcb-dri3.so.0 + 0x39ae)
#4  0x00000000f6f87a0c n/a (libGLX_nvidia.so.0 + 0x99a0c)
#5  0x00000000f6f6b1b9 n/a (libGLX_nvidia.so.0 + 0x7d1b9)
#6  0x00000000f6f3e1a4 n/a (libGLX_nvidia.so.0 + 0x501a4)
#7  0x00000000f6f3ebd1 n/a (libGLX_nvidia.so.0 + 0x50bd1)
#8  0x00000000f5b9de74 n/a (libnvidia-glcore.so.555.58.02 + 0xa9ce74)
#9  0x00000000f5b9e70b n/a (libnvidia-glcore.so.555.58.02 + 0xa9d70b)
#10 0x00000000f6f3a19b n/a (libGLX_nvidia.so.0 + 0x4c19b)
#11 0x00000000f6f3a94c n/a (libGLX_nvidia.so.0 + 0x4c94c)
#12 0x00000000f6fb4014 n/a (libGLX_nvidia.so.0 + 0xc6014)
ELF object binary architecture: Intel 80386
```

**Comment by scaronni** [comment]
Thanks!
